src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.link {
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+  --hover-color: rgba(0, 172, 220, 1);
+  display: block;
+}
+
+.link:hover {
+  color: var(--hover-color);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  list-style-type: none;
+  margin-left: 20px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.list {
+  display: flex;
+}
+
+.last {
+  margin-right: 50px;
+}
+
+.is-active {
+  color: var(--hover-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 37px;
```

Don't hardcode width, it should work for all links
```suggestion
  width: 100%;
```